docs/public/configuration/backend.md

```diff
@@ -319,6 +324,63 @@ Specifying a profile JSON file that contains no profile data is equivalent
 to specifying a profile JSON file containing the entire
 *Zonemaster Engine default profile*.
 
+
+## TLD URL section
+
+For the context of this section, see [TLD URL for GUI].
+
+The TLD URL section has several keys:
+* extract_url
+* maximum_lookup_time
+* suppress_log_messages
+* tld_\<TLD\> where `<TLD>` can be any string matching one of the following
+  patterns:
+  * `[a-z][a-z]+`
+  * `xn--[a-z0-9-][a-z0-9-]+`
+
+### extract_url
+
+Boolean value to enable the function described in document
+[TLD URL for GUI]. Disabling the function is the same as setting
+a global policy to block the function as described in the document.
+
+Accepted values: `yes` or `no`. Default to `yes` (enabled).
```

Comment:
I think it's good that we're converging on single values for true and false, but it think it would be better if we chose `true` and `false` rather than `yes` and `no`. That's more conventional and also in harmony with the literals used in the RPCAPI.

Reply:
See updates.